src/js/views/AppView.js

```diff
@@ -1,18 +1,92 @@
+import { emit } from '../utils/event.js';
 import { $ } from '../utils/dom.js';
 
+import Template from './Template.js';
+
 export default class AppView {
   constructor() {
+    this.template = new Template();
+
     this.$searchModalButton = $('#search-modal-button');
     this.$modalContainer = $('.modal-container');
+    this.$willSeeButton = $('#will-see-button');
+    this.$sawButton = $('#saw-button');
+    this.$willSeeWrapper = $('.will-see-wrapper');
+    this.$isEmptyWillSee = $('.is-empty-will-see', this.$willSeeWrapper);
+    this.$willSeeList = $('#will-see-list', this.$willSeeWrapper);
+    this.$sawWrapper = $('.saw-wrapper');
+    this.$isEmptySaw = $('.is-empty-saw', this.$sawWrapper);
+    this.$sawList = $('#saw-list', this.$sawWrapper);
 
     this.#bindEvents();
   }
 
   #bindEvents() {
-    this.$searchModalButton.addEventListener('click', this.#handleClickButton.bind(this));
+    this.$searchModalButton.addEventListener('click', this.#handleClickSearchButton.bind(this));
+    this.$willSeeButton.addEventListener('click', this.#handleClickWillSeeButton.bind(this));
+    this.$sawButton.addEventListener('click', this.#handleClickSawButton.bind(this));
+    this.$willSeeWrapper.addEventListener('click', this.#handleClickUserButton.bind(this));
+    this.$sawWrapper.addEventListener('click', this.#handleClickUserButton.bind(this));
   }
 
-  #handleClickButton() {
+  #handleClickSearchButton() {
     this.$modalContainer.classList.remove('hide');
   }
+
+  #handleClickWillSeeButton() {
+    this.$sawButton.classList.remove('clicked');
+    this.$willSeeButton.classList.add('clicked');
+
+    this.$sawWrapper.classList.add('hide');
+    this.$willSeeWrapper.classList.remove('hide');
+  }
+
+  #handleClickSawButton() {
+    this.$willSeeButton.classList.remove('clicked');
+    this.$sawButton.classList.add('clicked');
+
+    this.$willSeeWrapper.classList.add('hide');
+    this.$sawWrapper.classList.remove('hide');
+  }
+
+  #handleClickUserButton({ target }) {
+    if (target.className === 'user-delete-button') {
```

Comment:
`target`에 다른 class가 추가로 있다면 이 로직은 실패할 수 밖에 없을 것 같아요. [classList](https://developer.mozilla.org/ko/docs/Web/API/Element/classList)를 사용해서 변경에 취약하지 않게 구현해봐도 좋을 것 같아요 😄 
```suggestion
    if (target.classList.contains("user-delete-button")) {
```

Reply:
오오 이부분은 생각하지 못했어요! 다른 class가 생겨도 에러가 생기지 않도록, classList로 바꿔서 구현해보도록 하겠습니다. 감사합니다 👍